<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>b1MainWindow</class>
 <widget class="QMainWindow" name="b1MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>b1mulator</string>
  </property>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_state"/>
    <addaction name="actionLoad_state"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuEmulation">
    <property name="title">
     <string>&amp;Emulation</string>
    </property>
    <addaction name="actionPause"/>
    <addaction name="actionResume"/>
    <addaction name="actionStep"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionShowCPU"/>
    <addaction name="actionShowPPU"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEmulation"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open ROM</string>
   </property>
   <property name="toolTip">
    <string>Open ROM image</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionResume">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Resume</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>S&amp;tep 1 frame forward</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionShowCPU">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;CPU state</string>
   </property>
  </action>
  <action name="actionShowPPU">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;PPU state</string>
   </property>
  </action>
  <action name="actionSave_state">
   <property name="text">
    <string>Save state</string>
   </property>
  </action>
  <action name="actionLoad_state">
   <property name="text">
    <string>Load state</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>b1MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>b1MainWindow</receiver>
   <slot>openROM()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPause</sender>
   <signal>triggered()</signal>
   <receiver>b1MainWindow</receiver>
   <slot>pauseEmulation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionResume</sender>
   <signal>triggered()</signal>
   <receiver>b1MainWindow</receiver>
   <slot>resumeEmulation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStep</sender>
   <signal>triggered()</signal>
   <receiver>b1MainWindow</receiver>
   <slot>stepEmulation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_state</sender>
   <signal>triggered()</signal>
   <receiver>b1MainWindow</receiver>
   <slot>saveState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad_state</sender>
   <signal>triggered()</signal>
   <receiver>b1MainWindow</receiver>
   <slot>loadState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pauseEmulation()</slot>
  <slot>resumeEmulation()</slot>
  <slot>stepEmulation()</slot>
  <slot>saveState()</slot>
  <slot>loadState()</slot>
 </slots>
</ui>
